### PR TITLE
Add vshn mariadb backups

### PR DESCRIPTION
**Pull request by zugao**
## Summary

* This PR enables MariaDB customer backups via the apisserver

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Update tests.
- [x] Link this PR to related issues.

### CODE REVIEW

File: cmd/apiserver.go [hunk 1]
```diff
@@ -31,7 +32,8 @@ func newAPIServerCMD() *cobra.Command {
 		appcatEnabled = viper.GetBool("APPCAT_HANDLER_ENABLED")
 		vshnPGBackupsEnabled = viper.GetBool("VSHN_POSTGRES_BACKUP_HANDLER_ENABLED")
 		vshnRedisBackupsEnabled = viper.GetBool("VSHN_REDIS_BACKUP_HANDLER_ENABLED")
-		if !appcatEnabled && !vshnPGBackupsEnabled && !vshnRedisBackupsEnabled {
+		vshnMariaDBBackupEnabled = viper.GetBool("VSHN_MARIADB_BACKUP_HANDLER_ENABLED")
+		if !appcatEnabled && !vshnPGBackupsEnabled && !vshnRedisBackupsEnabled && !vshnMariaDBBackupEnabled {
```
Comment (Kidswiss) [inline cmd/apiserver.go:36]: This starts to not scale very well. Maybe we should come up with something better?

**Review by Kidswiss — APPROVED** [review]
It's a pity we can't make this more generic... a lot of copy-pasting happening here.

I just have one nitpick, I leave it up to you to change it or not.

**Comment by zugao** [comment]
> It's a pity we can't make this more generic... a lot of copy-pasting happening here.

I also feel the same. There is definitely way to abstract it further with functions or maybe even using types as arguments, the question is, do we really want to complicate things that much? There not so much code actually that is duplicated. We can have a brainstorming session if that's fine with you.

File: cmd/apiserver.go (hunk 1 above)
Comment (zugao) [reply cmd/apiserver.go:36]: Let me think about smth here

Comment (zugao) [reply cmd/apiserver.go:36]: The problem here is that we create the cmd object from the ApiServer instance. After the cmd is created there is no way to add Flags... Let's keep the current implementation and we come up with smth when we move away from the runtime repository.

Comment (Kidswiss) [reply cmd/apiserver.go:36]: Ah right, we're forced into that config library because of the runtime...

**Pull request merged**